Formula/netpbm.rb

```diff
@@ -21,6 +21,7 @@ class Netpbm < Formula
   depends_on "libtiff"
   unless OS.mac?
     depends_on "flex"
```

Comment:
`flex` is usually a build only dependency.
```suggestion
    depends_on "flex" =>: build
```